test/TEST-13-NSPAWN-SMOKE/test.sh

```diff
@@ -85,26 +84,29 @@ function run {
     fi
     if [[ "$2" = "yes" && "$is_cgns_supported" = "no" ]];  then
         printf "Cgroup namespaces are not supported. Skipping.\n"
+        return
     fi
 
-    export UNIFIED_CGROUP_HIERARCHY="$1"
-    export SYSTEMD_NSPAWN_USE_CGNS="$2"
-    local _root="/var/lib/machines/unified-$1-cgns-$2"
-    /create-busybox-container "$_root"
-    systemd-nspawn --register=no -D "$_root" -b
-    systemd-nspawn --register=no -D "$_root" --private-network -b
-    systemd-nspawn --register=no -D "$_root" -U -b
-    systemd-nspawn --register=no -D "$_root" --private-network -U -b
-    unset UNIFIED_CGROUP_HIERARCHY
-    unset SYSTEMD_NSPAWN_USE_CGNS
-}
+    set -x
+    # Call sleep before each container to make the logs easier to parse
+    systemd-run -E UNIFIED_CGROUP_HIERARCHY=$1 -E SYSTEMD_NSPAWN_USE_CGNS=$2 -E SYSTEMD_LOG_LEVEL=debug \
+                --wait systemd-nspawn --quiet --register=no -M container -b
```

Comment:
Anyway, `systemd-run` is a complex thing. `systemd-run --wait` is a complex thing. And I really want to test `systemd-nspawn` (not `systemd-run`, not `machined` (this is why `--register=no`)).